packages/host/app/components/search-sheet/index.gts

```diff
@@ -73,22 +78,27 @@ let elementCallback = modifier(
 );
 
 export default class SearchSheet extends Component<Signature> {
-  @tracked searchKey = '';
-  @tracked isSearching = false;
-  searchCardResults: CardDef[] = new TrackedArray<CardDef>();
+  @tracked private searchKey = '';
+  @tracked private isSearching = false;
+  private searchCardResults: CardDef[] = new TrackedArray<CardDef>();
   @tracked cardURL = '';
   @service declare operatorModeStateService: OperatorModeStateService;
   @service declare cardService: CardService;
   @service declare loaderService: LoaderService;
   @service declare recentCardsService: RecentCards;
 
-  get inputBottomTreatment() {
+  constructor(owner: Owner, args: any) {
+    super(owner, args);
+    this.args.onSetup(this.doExternallyTriggeredSearch);
+  }
+
+  private get inputBottomTreatment() {
     return this.args.mode == SearchSheetModes.Closed
       ? BoxelInputBottomTreatments.Rounded
       : BoxelInputBottomTreatments.Flat;
   }
 
-  get searchLabel() {
+  private get searchLabel() {
```

Comment:
a more user friendly thing we could do here is to actually show the display name for the card rather than the code-ref URL. However that wasn't actually in the requirements for this ticket and would require importing and running the card def which seemed probably overkill for this PR (we'd need to incorporate a resource to load the card def when the search type is a code ref). Perhaps as a followup if we decide we want to do that we can add that feature